modules/virt-adding-a-boot-source-web.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-adding-a-boot-source-web_{context}"]
+= Adding a boot source for a virtual machine template
+
+A boot source must be configured for any virtual machine template that you want to use for creating virtual machines or custom templates. When  virtual machine templates are configured with a boot source, they are labeled *Available* in the *Templates* tab.
```

Comment:
```suggestion
A boot source can be configured for any virtual machine template that you want to use for creating virtual machines or custom templates. When  virtual machine templates are configured with a boot source, they are labeled *Available* in the *Templates* tab.
```
admin users can create a virtual machine from a template that as no source attached